### PR TITLE
Added set-wallpaper and set-screensaver

### PR DESCRIPTION
**Pull request by sethbrasile**

### CODE REVIEW

File: Function.Set-Screensaver.ps1 [hunk 1]
```diff
@@ -0,0 +1,131 @@
+Function Set-Screensaver {
```
Comment (wmmatt) [inline Function.Set-Screensaver.ps1:1]: Would like to have more standard function docuemntation on this like the .PARAMETER xx format, and the .EXAMPLE xx format. You have a lot of comments on the function in general so should be fine for now, but maybe hit this with a #TODO where the documentation would be.

**Review by wmmatt — APPROVED** [review]
These are awesome...nice work

File: Function.Set-Screensaver.ps1 (hunk 1 above)
Comment (sethbrasile) [reply Function.Set-Screensaver.ps1:1]: 👍🏻 will do

**Comment by sethbrasile** [comment]
will be adjusting to handle all users

File: Function.Set-Screensaver.ps1 (hunk 1 above)
Comment (sethbrasile) [reply Function.Set-Screensaver.ps1:1]: Done

**Review by wmmatt — APPROVED** [review]
Clean. Looks good!

**Comment by dkattan** [comment]
Are you sure this works if no one is logged into the machine?

The SIDs you see when enumerating HKU are those mapped to ntuser.dat when a user logs in. They will sometimes remain mounted even if the user logs out. Restart a computer, don’t login, and run Get-ChildItem HKU:\ and you won’t get anything.

You may want to have a look at Invoke-HKCU in Immy, or reference how PDQ does it: https://www.pdq.com/blog/modifying-the-registry-users-powershell/

**Comment by sethbrasile** [comment]
I'm not sure @dkattan, I bet you're right. I hate Windows with the entirety of my being.

**Pull request merged**